### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by asharanjith**
In the session, I add a desktop version of the Portfolio website

- Create two CSS files for adding media queries for mobile and desktop version
- Update index.html with necessary changes in its content.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,37 +8,53 @@
...
+          <a href="#" class="logo">My logo</a>
+          <i class="fas fa-bars" id="hamburger"></i>
+          
+          <ul class="nav-list">
+            
+            <li class="nav-items">Portfolio</li>
+            <li class="nav-items">About</li>
+            <li class="nav-items">Contact</li>
+          </ul>
+        </nav>
```
Comment (christianbiring1) [inline index.html:27]: - [x] Kindly make sure your design sticks to the Figma design as much as possible. 
Figma design :point_down: 
image
Your design :point_down: 
image

File: mobile.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  .Introduction {
+    background: url(Assets/header.png) no-repeat center, #fff;
+    background-size: contain;
+    padding: 10px;
+  }
+
+  #hamburger {
+    align-self: flex-end;
+  }
```
Comment (christianbiring1) [inline mobile.css]: - [x] Kindly make sure you start with the mobile approach as it's the standard. `@media screen and (min-width: 768px)`

File: styles.css [hunk 3]
```diff
@@ -182,10 +136,9 @@ body {
 }
 
 .subtitle-container {
-  width: 100%;
+  width: 110%;
   font-size: 13px;
   font-weight: 600;
-  line-height: 16px;
   margin-top: 0;
   border-top: 0;
```
Comment (christianbiring1) [inline styles.css]: - [ ] KIndly make sure your page doesn't overflow as you can see here on your design for the mobile version :point_down: 
image

File: mobile.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+  .nav-bar {
+    padding: 10px;
+  }
+
+  .Introduction {
+    background: url(Assets/header.png) no-repeat center, #fff;
+    background-size: contain;
+    padding: 10px;
+  }
```
Comment (christianbiring1) [inline mobile.css]: - [ ] Kindly ensure you use the correct typo `font`, `color` as the template provided in Figma.
Your template :point_down: 
image
The Figma design :point_down: 
image

File: mobile.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+  .langs {
+    display: flex;
+    margin-top: 5px;
+    flex-direction: column;
+  }
+
+  .about-content {
+    display: flex;
+    flex-direction: column;
```
Comment (christianbiring1) [inline mobile.css]: - [x] Kindly make sure the images are visible on the screen size of `768px`. As you can see here your design
image

File: index.html [hunk 6]
```diff
@@ -261,7 +311,7 @@ <h4 class="media-text">LET’S CONNECT</h4>
                                 <li><a href="#" class="skill-link">Skills</a></li>
                             </ul>
                         </div>
-                        <div class="v-icon">
+                        <div class="v-icon rotate">
```
Comment (christianbiring1) [inline index.html:314]: - [ ] Kindly make sure you stick to the original design provided in Figma. the `typo:(font-size)`, the `width` and `height` of the icons must match.
As you can see your design :point_down: 
image
Figma design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @asharanjith , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :tada: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`

## Required Changes ♻️
- [ ] Kindly make sure your page is responsive on all the screen size
- [ ] Kindly make sure the mobile-first method is used
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: desktop.css [hunk 7]
```diff
@@ -0,0 +1,106 @@
+@media screen and (min-width: 768px) {
+  #hamburger {
+    display: none;
+  }
```
Comment (jerryowusu) [inline desktop.css:4]: Your project is looking good. :+1:  The only issue I have is that the design does not conform to the desktop figma design between screen width of 768px and 1041px. Check the images below :point_down:  for illustration. Kindly ensure that it conforms to the dimensions of the desktop figma design for all screen size greater that 768px as stated in the project requirements. For your project to be responsive it has to conform to the figma designs in all screen sizes

**Headline Section**

Your design             |  Figma design
:-------------------------:|:-------------------------:
Screenshot from 2022-09-30 15-26-02  |  Screenshot from 2022-09-30 15-27-42

Your design             |  Figma design
:-------------------------:|:-------------------------:
Screenshot from 2022-09-30 15-26-17  |  Screenshot from 2022-09-30 15-27-57

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @asharanjith 

Great work on making the changes requested by a previous reviewer 👏 . You are almost there. :wink: You've done well implementing some of the requested changes, but there is one more issue which isn't addressed yet (_This was requested by the previous reviewer_).

almost there


 ## Highlights
All linters are passing :heavy_check_mark:
Correct GitHub workflow :heavy_check_mark:
Meaningful commit messages :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 8]
```diff
@@ -240,22 +177,19 @@ body {
...
-  line-height: 16px;
   letter-spacing: 0.03em;
   background-color: #d9d9f5;
-  border-radius: 9px;
-  opacity: 0.5;
+  border-radius: 5px;
   color: #6070ff;
-  padding: 4px 12px;
+  flex-grow: 0;
+  padding: 0 4px 0 4px;
```
Comment (tugsem) [inline styles.css:185]: - [ ] Kindly fix word wrappings in your cards to match the figma design. 
image

File: styles.css [hunk 9]
```diff
@@ -113,79 +90,49 @@ body {
...
 }
 
 .subtitle-container {
-  width: 100%;
   font-size: 13px;
   font-weight: 600;
-  line-height: 16px;
   margin-top: 0;
   border-top: 0;
 }
```
Comment (tugsem) [inline styles.css:138]: - [ ] In your 'subtitle-items' class, please disable word-wrapping and use flexbox to align items vertically. That would look more professional.
image

**Review by tugsem — APPROVED** [review]
Hi @asharanjith , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights ⭐ 
✅ Nice work doing requested changes
## Optional suggestions

- I highly suggest you to apply the changes under the review. Previous reviewer didn't request them clearly but you need them for increasing your recruitment chance in the future.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 9 above)
Comment (asharanjith) [reply styles.css:138]: @tugsem Thank you. Will work on the suggestions and fix them.

**Pull request merged**